### DISCUSSION TITLE
Set animation in AR mode

### DISCUSSION THREAD

**Discussion by Gianlupedro73** [google/model-viewer#2048] [Q&A]
Hello, I have a glb model with 4 or 5 animations. I have for example "run" and "roar"... 
I setted animation-name="roar" on "browser view", but when I start AR mode on my phone the animation is always "run". 
is it possible to set the animation also in AR mode? 
Thanx

**Comment by elalish** [comment]
Only if you use WebXR mode (this is part of why it's our default). Of course this only works on Chrome Android for now. We have very little ability to customize the native AR apps.

**Reply by Gianlupedro73** [reply]
Sorry, I have not clear what are you saying.
This is my code, and in this way the animation "roar" works just on web browser.
On my Android phone with chrome, when I click AR icon it open AR visualization with "run" animation

<model-viewer 
        src="vibrant_T-Rex.glb" 
        ios-src="vibrant_T-Rex.usdz"
        ar 
        ar-modes="scene-viewer webxr quick-look" 
        ar-scale="auto"
        animation-name="roar" 
        camera-controls 
        shadow-intensity="3" 
        shadow-softness="1" 
        autoplay 
        poster="vibrant_T-Rex.png"
        auto-rotate
        ar>

</model-viewer>

**Reply by elalish** [reply]
Try restricting [ar-modes](https://modelviewer.dev/docs/#entrydocs-augmentedreality-attributes-arModes) to one value at a time to be sure which mode you're in. What device/browser version are you using? Not all modes are available everywhere.
